lib/smart_proxy_dynflow/plugin.rb

```diff
@@ -4,29 +4,18 @@
 
 class Proxy::Dynflow
   class Plugin < Proxy::Plugin
-    rackup_path = begin
-      require 'smart_proxy_dynflow_core'
-      'http_config_with_executor.ru'
-    rescue LoadError
-      'http_config.ru'
-    end
-    http_rackup_path File.expand_path(rackup_path, File.expand_path("../", __FILE__))
-    https_rackup_path File.expand_path(rackup_path, File.expand_path("../", __FILE__))
+    rackup_path = File.expand_path('http_config.ru', File.expand_path("../", __FILE__))
```

Comment:
Isn't this equvalent to:
```suggestion
    rackup_path = File.expand_path('http_config.ru', __dir__)
```

Reply:
Good point, this is a remnant of supporting ruby `< 2.0`. Will change